documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,434 @@
+* Validate permissions on source providers link:https://issues.redhat.com/browse/MTV-273[(MTV-273)]
+* Validation for RHEL 6 link:https://issues.redhat.com/browse/MTV-413[(MTV-413)]
+* Integrate volume populators in remote migrations link:https://issues.redhat.com/browse/MTV-721[(MTV-721)]
+* Custom mapping of instance type preferences link:https://issues.redhat.com/browse/MTV-453[(MTV-453)]
+* VM posted with `Running` state link:https://issues.redhat.com/browse/MTV-794[(MTV-794)]
+* Review and configure migrated VMs link:https://issues.redhat.com/browse/MTV-843[(MTV-843)]
+* Default virt storage class link:https://issues.redhat.com/browse/MTV-855[(MTV-855)]
+
+.Replace templates with flavors
+
+Kubevirt API introduces `VirtualMachineClusterFlavor` and `VirtualMachineFlavor` CDIs. Flavors are meant to replace Template, which is only available on {ocp-name}. link:https://issues.redhat.com/browse/MTV-206[(MTV-206,] link:https://issues.redhat.com/browse/MTV-919[MTV-919)]
+
+.Integrate volume populators in remote migrations
+// Arik, please check and explain
+Integrate volume populator for `oVirt` and OpenStack, together with `virt-v2v` on `el9` into a CDI to migrate remote {ocp-name} clusters. link:https://issues.redhat.com/browse/MTV-721[(MTV-721)]
+
+.VM posted with `Running` state
+
+{project-short} shows each VM's migration `Running` state during the migration. link:https://issues.redhat.com/browse/MTV-794[(MTV-794)]
+
+.Review and configure migrated VMs
+
+The {project-short} web console shows the name and migration status of each VM. link:https://issues.redhat.com/browse/MTV-843[(MTV-843)]
+
+.Default virt storage class
+
+For requests that do not specify an explicit storage class, the `is-default-virt-class` replaces the Kubernetes default storage class. link:https://issues.redhat.com/browse/MTV-855[(MTV-855)]
+
+.Validate permissions on source providers
+
+When migrating a VM with an RHEV user with restricted permissions, {project-short} validates user permissions and the migration pod reports where the process fails. link:https://issues.redhat.com/browse/MTV-273[(MTV-273)]
+
+.Validation for RHEL 6
+
+Enterprise Linux 9 does not support CNV to provide RHEL 6 templates. This restriction is no longer relevant because `VirtualMachineClusterFlavor` CRDs replaced template. link:https://issues.redhat.com/browse/MTV-413[(MTV413)]
+
+.Custom mapping of instance type preferences
+
+{project-short} enables users to add Customer Virtual Machine Mapping, which are custom templates that configured to specific migrations, VMs, and users. link:https://issues.redhat.com/browse/MTV-453[(MTV-453)]
+// 
+// .CNV InstanceTypes [MTV-919]
+// This looks like it repeats MTV-206 above
+
+.RH-SDL
+
+{project-short} conforms with the Red Hat Secure Development Lifecycle (RH-SDL) that allows users to follow an implementation plan that adopts security controls during the product lifecycle. link:https://issues.redhat.com/browse/MTV-801[(MTV-801)]
```

Comment:
```suggestion
{project-short} conforms with the Red Hat Secure Development Lifecycle (RH-SDL) that allows users to follow an implementation plan, which adopts security controls during the product lifecycle. link:https://issues.redhat.com/browse/MTV-801[(MTV-801)]
```

your senstence structure is bad: `... that ... that... `

Reply:
@ahadas 
See links below to Red hat internal articles.
Do we have these articles in publicly available sites?